dpnp/dpnp_iface_indexing.py

```diff
@@ -141,6 +142,8 @@ def choose(x1, choices, out=None, mode="raise"):
             pass
         elif not choices_list:
             pass
+        elif dpnp.is_cuda_backend(x1):
+            pass
```

Comment:
As discussed, it would be better to place explicit `NotImplementedError` rather than falling back on numpy here.